services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -78,6 +86,43 @@ const Asset = ({ libraryData }) => {
     description
   }
 
+  const { sponsor } = assetData.params
+  const SponsorIcon = teams[sponsor] && teams[sponsor].icon
+
+  const AssetItemMeta = ({ properties }) => {
+    const renderStatus = () => {
+      const { name: assetStatus } = status[assetData.content.status]
+
+      if (!assetStatus) return null
+
+      return (
+        <>
+          <StatusIcon className={dashboardStyles.metaIcon} status={assetData.content.status} />
```

Comment:
These styles seem specific to this page. I'd move them to a SCSS module specific to this page.